Sources/Core/AWSClientRuntime/UserAgent/OSMetadata.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+
+struct OSMetadata {
+    let family: PlatformOperatingSystem
+    let version: String?
+
```

Comment:
In SEP, there's optional (0 to infinity) additional metadata appended to the end of the OS metadata. Is there a reason for the discrepancy?

Reply:
`AdditionalMetadata` has been added to `OSMetadata`.